src/crypto/Signer.h

```diff
@@ -35,24 +34,30 @@ namespace digidoc
           virtual std::vector<unsigned char> sign(const std::string &method, const std::vector<unsigned char> &digest) const = 0;
           std::string method() const;
           std::string profile() const;
+          bool useENProfile() const;
 
           std::string city() const;
+          std::string streetAddress() const;
           std::string stateOrProvince() const;
           std::string postalCode() const;
           std::string countryName() const;
           std::vector<std::string> signerRoles() const;
           void setMethod(const std::string &method);
           void setProfile(const std::string &profile);
+          void setENProfile(bool use);
```

Comment:
I would suggest renaming:
- setENProfile() to enableENProfile() (or toggleENProfile(), as the comment on it actually suggests)
- useENProfile() to isENProfileEnabled()
Right now "set" and "use" are similar verbs and it could be clearer on which does what